bitnami/ejbca/values.yaml

```diff
@@ -302,6 +323,52 @@ ingress:
   ##
   annotations: {}
 
+  ## Enable TLS configuration for the hostname defined at ingress.hostname parameter
+  ## TLS certificates will be retrieved from a TLS secret with name: {{- printf "%s-tls" .Values.ingress.hostname }}
+  ## You can use the ingress.secrets parameter to create this TLS secret or relay on cert-manager to create it
+  ##
+  tls: false
+
+  ## The list of additional hostnames to be covered with this ingress record.
+  ## Most likely the hostname above will be enough, but in the event more hosts are needed, this is an array
+  ## extraHosts:
+  ## - name: ejbca.local
+  ##   path: /
+  ##
+
+  ## Any additional arbitrary paths that may need to be added to the ingress under the main host.
+  ## For example: The ALB ingress controller requires a special rule for handling SSL redirection.
+  ## extraPaths:
+  ## - path: /*
+  ##   backend:
+  ##     serviceName: ssl-redirect
+  ##     servicePort: use-annotation
+  ##
```

Comment:
Not sure if we are adding an empty comment `##` at the end of a block or we usually finish the commented blocks without this

Reply:
Check for example:
```yaml
## String to partially override ebjca.fullname template (will maintain the release name)
##
# nameOverride:

## String to fully override ebjca.fullname template
##
# fullnameOverride:
```
is not
```yaml
## String to partially override ebjca.fullname template (will maintain the release name)
##
# nameOverride:
##

## String to fully override ebjca.fullname template
##
# fullnameOverride:
##
```